peachjam/forms.py

```diff
@@ -107,8 +108,11 @@ def filter_queryset(self, queryset, exclude=None):
         if alphabet and exclude != "alphabet":
             queryset = queryset.filter(title__istartswith=alphabet)
 
-        if author and exclude != "author":
-            queryset = queryset.filter(author__name__in=author)
+        if authors and exclude != "author":
+            queryset = queryset.filter(author__name__in=authors)
+
+        if courts and exclude != "courts":
+            queryset = queryset.filter(court__name__in=courts)
 
```

Comment:
@longhotsummer Not sure if this code is working as expected. Should it narrow out the facet list after filtering on a parameter or not?

Reply:
> Migration worked well for me 👍
> 
> Please also add Court as a filter and a column for judgments in the admin view.

Missed this 😢 Let me add now.

Reply:
> 

Done

Reply:
> @longhotsummer Not sure if this code is working as expected. Should it narrow out the facet list after filtering on a parameter or not?

You've got it doing the correct thing. It applies the courts filter unless explicitly told to exclude it.